app/models/docker_builder_service.rb

```diff
@@ -60,6 +61,7 @@ def push_image(tag)
         build.docker_repo_digest = "#{project.docker_repo}@#{matches[1]}"
       end
     end
+    build.docker_repo_digest ||= result.info['RepoTags'].first
```

Comment:
Aye, I agree with sentiment. But don't want to change the column name in this PR.